build.gradle

```diff
@@ -10,7 +10,7 @@ buildscript {
         opensearch_group = "org.opensearch"
         isSnapshot = "true" == System.getProperty("build.snapshot", "true")
         opensearch_version = System.getProperty("opensearch.version", "2.0.0-rc1-SNAPSHOT")
```

Comment:
does this also need to be changed?

Reply:
I had the same concern. This is not included in the initial request from infra team. Just confirmed with them and looks like this needs to be updated to "2.0.0-SNAPSHOT"

Reply:
2.0.0 snapshot is ready. updated to use version of 2.0.0-SNAPSHOT.